app/views/flights/_flight.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="flight-<%= flight[:flight].id %>">
+  <h2>Flight Number: <%= flight[:flight].number %></h2>
+  <p>Airline: <%= flight[:flight].airline_name %></p>
+  <p>Passengers:</p>
+  <ul>
+    <% flight[:passengers].each do |passenger| %>
+      <li>
+        <%= passenger[:name] %>
+        <%= link_to 'Remove', flight_flight_passenger_path(flight[:flight].id, passenger[:id]), method: :delete %>
+      </li>
+    <% end %>
+  </ul>
+</div>
```

Comment:
This US is not implemented properly. I have not taken a look at the controller yet, but just looking at this code, there is no reason to have to dig into a variable to pull out a key.

Your path for you button is also incorrect.